component.css

```diff
@@ -18,6 +21,9 @@
 h1,h2,h3,h4,h5,h6{
     color: var(--white);
```

Comment:
instead of setting colours for each (text) element, you can set font colour in the body tag directly.